### ISSUE TITLE
ibm_cis_dns_record proxied does not work

### ISSUE THREAD

**Issue by ifs-pauljegouic** [IBM-Cloud/terraform-provider-ibm#1248]
### Terraform Version
```bash
➜  tmp terraform -v
Terraform v0.12.23
+ provider.ibm v1.2.6
```

### Affected Resource(s)
Please list the resources as a list, for example:
- ibm_cis_dns_record

### Terraform Configuration Files
```hcl
provider "ibm" {
  region = "eu-de"
}

resource "ibm_cis_dns_record" "custom_ingress" {
  cis_id = MYCIS_ID
  domain_id = MYDOMAIN_ID
  name      = "foobar"
  type      = "CNAME"
  content   = "myurl.com"
  proxied   = true
}
```

### Debug Output
```hcl
➜  tmp terraform plan 
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.


------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # ibm_cis_dns_record.custom_ingress will be created
  + resource "ibm_cis_dns_record" "custom_ingress" {
      + cis_id      = "crn:v1:bluemix:public:internet-svcs:global:a/9e87f2cde0fe48ee9e056bdad79d285d:ad5b006d-0c7a-40ff-94e6-0f6a9a77ef7c::"
      + content     = "featuresk277-e2e-529811-bae11403bfb1a9569e490cfdf7fda222-0000.eu-de.containers.appdomain.cloud"
      + created_on  = (known after apply)
      + domain_id   = "19364270fc616b800a9e37f940fd292e:crn:v1:bluemix:public:internet-svcs:global:a/9e87f2cde0fe48ee9e056bdad79d285d:ad5b006d-0c7a-40ff-94e6-0f6a9a77ef7c::"
      + id          = (known after apply)
      + modified_on = (known after apply)
      + name        = "testpaul3"
      + proxiable   = (known after apply)
      + proxied     = true
      + type        = "CNAME"
    }

Plan: 1 to add, 0 to change, 0 to destroy.
```

```hcl
➜  tmp terraform apply

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # ibm_cis_dns_record.custom_ingress will be created
  + resource "ibm_cis_dns_record" "custom_ingress" {
      + cis_id      = "crn:v1:bluemix:public:internet-svcs:global:a/9e87f2cde0fe48ee9e056bdad79d285d:ad5b006d-0c7a-40ff-94e6-0f6a9a77ef7c::"
      + content     = "featuresk277-e2e-529811-bae11403bfb1a9569e490cfdf7fda222-0000.eu-de.containers.appdomain.cloud"
      + created_on  = (known after apply)
      + domain_id   = "19364270fc616b800a9e37f940fd292e:crn:v1:bluemix:public:internet-svcs:global:a/9e87f2cde0fe48ee9e056bdad79d285d:ad5b006d-0c7a-40ff-94e6-0f6a9a77ef7c::"
      + id          = (known after apply)
      + modified_on = (known after apply)
      + name        = "foobar"
      + proxiable   = (known after apply)
      + proxied     = true
      + type        = "CNAME"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

ibm_cis_dns_record.custom_ingress: Creating...
ibm_cis_dns_record.custom_ingress: Creation complete after 2s [id=d44c82d5782763b350c45e516ea8ba14:19364270fc616b800a9e37f940fd292e:crn:v1:bluemix:public:internet-svcs:global:a/9e87f2cde0fe48ee9e056bdad79d285d:ad5b006d-0c7a-40ff-94e6-0f6a9a77ef7c::]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

### Expected Behavior
The CIS DNS Record is actually proxied

### Actual Behavior
Not proxied

2nd `terraform apply`: 

```hcl
➜  tmp terraform plan 
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

ibm_cis_dns_record.custom_ingress: Refreshing state... [id=d44c82d5782763b350c45e516ea8ba14:19364270fc616b800a9e37f940fd292e:crn:v1:bluemix:public:internet-svcs:global:a/9e87f2cde0fe48ee9e056bdad79d285d:ad5b006d-0c7a-40ff-94e6-0f6a9a77ef7c::]

------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # ibm_cis_dns_record.custom_ingress will be updated in-place
  ~ resource "ibm_cis_dns_record" "custom_ingress" {
        cis_id      = "crn:v1:bluemix:public:internet-svcs:global:a/9e87f2cde0fe48ee9e056bdad79d285d:ad5b006d-0c7a-40ff-94e6-0f6a9a77ef7c::"
        content     = "featuresk277-e2e-529811-bae11403bfb1a9569e490cfdf7fda222-0000.eu-de.containers.appdomain.cloud"
        created_on  = "2020-03-30T15:49:48.289461Z"
        data        = {}
        domain_id   = XXXX
        id          = XXXX
        modified_on = "2020-03-30T15:49:48.289461Z"
        name        = "foobar"
        priority    = 0
        proxiable   = true
      ~ proxied     = false -> true
        type        = "CNAME"
    }

Plan: 0 to add, 1 to change, 0 to destroy.

------------------------------------------------------------------------
```

Even in the UI console we can see that the parameter is not taken in account.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`
2. `terraform apply`
3. `terraform plan`

**Comment by ifs-pauljegouic** [comment]
@hkantare seeing a lot of commit :) :) 

When can we expect this feature to be released ?

**Comment by hkantare** [comment]
planning for a release mostly by today EOD it will be part of the release

**Comment by ifs-pauljegouic** [comment]
Fixed with 1.4.0. Thank you
